### PR TITLE
rm deprecated template and add UUID validation on disk_image

### PR DESCRIPTION
**Pull request by uzaxirr**
Screenshot 2024-08-01 at 2 34 38 PM

### CODE REVIEW

**Comment by uzaxirr** [comment]
Screenshot 2024-08-06 at 12 44 55 PM

File: internal/utils/utils.go [hunk 1]
```diff
@@ -240,3 +241,13 @@ func ParseErrorResponse(errorMsg string) (*CustomError, error) {
 	}
 	return &customErr, nil
 }
+
+// ValidateUUID checks if a given string is a UUID or not
+func ValidateUUID(v interface{}, k string) (ws []string, errors []error) {
```
Comment (RealHarshThakur) [inline internal/utils/utils.go:246]: I don't understand this function signature. Why are v and ws necessary? 
We could accomplish the same with k and returning errors.

Comment (RealHarshThakur) [reply internal/utils/utils.go:246]: is it because the validate func expects that signature?

Comment (RealHarshThakur) [reply internal/utils/utils.go:246]: (answer was yes)

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**